content/docs/hello-world.md

```diff
@@ -8,11 +8,8 @@ next: introducing-jsx.html
 
 The smallest React example looks like this:
 
-```js
-ReactDOM.render(
-  <h1>Hello, world!</h1>,
-  document.getElementById('root')
-);
+```jsx
+<h1>Hello, world!</h1>
```

Comment:
Nice to have:
```suggestion
ReactDOM.createRoot(document.getElementById('root')).render(<h1>Hello, world!</h1>);
```

Preview link: https://reactjs-org-7ht1fcs9w-fbopensource.vercel.app/docs/hello-world.html
Docs: https://reactjs.org/docs/hello-world.html

Reply:
Thanks!